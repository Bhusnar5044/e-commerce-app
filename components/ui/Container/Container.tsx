import { memo } from 'react';

interface ContainerProps {
    children: React.ReactNode;
}

export const Container: React.FC<ContainerProps> = memo(({ children }) => {
    return <div className="mx-auto max-w-screen-2xl">{children}</div>;
});

Container.displayName = 'Container';
