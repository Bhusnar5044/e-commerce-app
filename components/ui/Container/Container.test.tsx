import { render } from '@test-utils';
import { Container } from './Container';

describe('Container', () => {
    it('should render properly', () => {
        const { container } = render(<Container>dummy</Container>);
        expect(container).toMatchSnapshot();
    });
});
