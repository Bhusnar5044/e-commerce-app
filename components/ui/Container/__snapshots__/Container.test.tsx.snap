// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render properly 1`] = `
<div>
  <div
    class="mx-auto max-w-screen-2xl"
  >
    dummy
  </div>
</div>
`;
